README.md

```diff
@@ -154,23 +154,17 @@ profile_metrics_alerting::tools::whitelabel_subtitle: "Managed by ORGANIZATION G
 profile_metrics_alerting::tools::whitelabel_title: "Monitoring and Telemetry Interface"
 ```
 
-To pause/start alerts on a schedule (applicable for old style alerts only), you can use these parameters:
+To pause/start alerts on a schedule you can use these parameters:
 ```yaml
-# Format for command is similar to running alert_toggle.sh by hand, see README in files/root/grafana_tools
-profile_metrics_alerting::alert_cycle::crons:
-   "Pause mforgehn1 alerts":
-    command: "pause tag \"mforgehn1\""
-    hour: 21
+# Will add a 45 min silence for any alert with a host label of mforgehn1.*
+# See this repo for details on arguments for command: https://git.ncsa.illinois.edu/ici-monitoring/grafana-tools
+  "Silence mforgehn1 alerts (label=host)":
+    command: "auto_silence_root_cron \"Silence mforgehn1 alerts (label=host)\" host =~ \"mforgehn1.*\" now \"+45 min\""
```

Comment:
Where does the `auto_silence_root_cron` command come from?

Ah, that is a parameter to the actual `/root/grafana_tools/bulk_alert/unified_alerts/silence.sh` script. That gets overridden via https://github.com/ncsa/puppet-profile_metrics_alerting/blob/bd44a91fba0f2e4ada3a4eaa4d4d8d9cebc97150/manifests/alert_cycle.pp#L23-L31

Is that correct? This is confusing just seeing it in the review out of context.

Reply:
The purpose of the code you quoted is to inject the script path (saved as `profile_metrics_alerting::alert_cycle::script_path`) in front of each command someone gives in `profile_metrics_alerting::alert_cycle::crons:`.

That way when someone makes the cron they don't have to add `/root/grafana_tools/bulk_alert/unified_alerts/silence.sh ` in front every time. Also makes it easy when the script changes.

The `auto_silence_root_cron` does end up in the command, the first argument to that script is the username of the person making the silence. So I wanted to pick a name that indicated it was coming from cron.